test/uk/gov/hmrc/vatsubscription/helpers/WhyTurnoverBelowTestConstants.scala

```diff
@@ -37,11 +38,12 @@ object WhyTurnoverBelowTestConstants {
     "reducedTradingHours" -> true,
     "seasonalBusiness" -> true,
     "closedPlacesOfBusiness" -> true,
-    "turnoverLowerThanExpected" -> true
+    "turnoverLowerThanExpected" -> true,
+    "alreadyBelow" -> true
   )
 
   val whyTurnoverBelowString: String =
```

Comment:
Should we gave a test where it's `None` and the only output is "Already below the threshold"?

Reply:
Isn't this covered in the TurnoverBelowThresholdSpec?

Reply:
Yep true